packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -312,6 +331,16 @@ class SemanticsNode extends AbstractNode {
     }
   }
 
+  Set<SemanticsTag> _tags = new Set<SemanticsTag>();
+
+  /// Tag the [SemanticsNode] with [tag].
+  void addTag(SemanticsTag tag) {
```

Comment:
What if you added it before, and now you want to unadd it? Like setting checked to true, then setting it to false.